components/list_item/list_item.vue

```diff
@@ -224,14 +243,15 @@ export default {
   border-radius: var(--size-300);
 }
 
-.dt-list-item--focusable:focus,
-.dt-list-item--focusable:focus-within,
```

Comment:
Shouldn't the selected variant re-style the background for these states rather than removing them from the default list item?

Reply:
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/89984179/217563629-8440fce6-692a-417a-8830-1d726c3aa79b.png">

<img width="1266" alt="image" src="https://user-images.githubusercontent.com/89984179/217563968-6c31ad54-1acb-4480-aaca-76b6d240ae6d.png">

WDYT now? :)